addons/binding/org.openhab.binding.nibeuplink/src/main/java/org/openhab/binding/nibeuplink/internal/connector/UplinkWebInterface.java

```diff
@@ -156,9 +162,8 @@ public void update(CommunicationStatus status) {
      *
      * @param config the Bridge configuration
      */
-    public UplinkWebInterface(NibeUplinkConfiguration config, ScheduledExecutorService scheduler,
-            NibeUplinkHandler handler, HttpClient httpClient) {
-        this.config = config;
+    public UplinkWebInterface(ScheduledExecutorService scheduler, NibeUplinkHandler handler, HttpClient httpClient) {
+        this.config = handler.getConfiguration();
```

Comment:
Is this allocation necessary? I expect the `start()` method called later anyways which accesses the configuration again.

Reply:
Well otherwise I would have to check for null on each access. I do not think that this would be better.